src/Core/View.js

```diff
@@ -224,6 +229,9 @@ View.prototype.addLayer = function addLayer(layer, parentLayer) {
 View.prototype.notifyChange = function notifyChange(needsRedraw, changeSource) {
     this._changeSources.add(changeSource);
     this.mainLoop.scheduleViewUpdate(this, needsRedraw);
+    if (needsRedraw) {
+        this.dispatchEvent({ type: VIEW_EVENTS.UPDATED });
```

Comment:
1. I'm wary of using events to *synchronize* things. Are you sure your listener will be called in a sync manner? Or will they be called after the update is performed? or after the rendering? or "it depends"?
2. The name and usage of your event disagree with each other:
```
itowns.GLOBE_VIEW_EVENTS.UPDATED, isDisplayed
```
(note that there's already a way to plug a function just before rendering: `view.preRender`)